diff --git a/themes/af-magic.zsh-theme b/themes/af-magic.zsh-theme
index 1c6d173..cdf902c 100644
--- a/themes/af-magic.zsh-theme
+++ b/themes/af-magic.zsh-theme
@@ -8,7 +8,6 @@ local return_code="%(?..%{$fg[red]%}%? ↵%{$reset_color%})"
 # primary prompt
 PROMPT='$FG[237]------------------------------------------------------------%{$reset_color%}
 $FG[032]%~\
-$(git_prompt_info) \
 $FG[105]%(!.#.»)%{$reset_color%} '
 PROMPT2='%{$fg[red]%}\ %{$reset_color%}'
 RPS1='${return_code}'
